sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/AzureSecurityKeyVaultAdministrationModelFactory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary> Model factory for AzureSecurityKeyVaultAdministration read-only models. </summary>
```

Comment:
The name in the comment doesn't make sense. I'd just elide it from generated code and leave the comment "Model factory for read-only models." The package name really doesn't add much value to the statement anyway.